build.sh

```diff
@@ -118,6 +118,17 @@ write_archive_info() {
     prepare_git_artifacts "${srcdir}" /cosa/coreos-assembler-git.tar.gz /cosa/coreos-assembler-git.json
 }
 
+libguestfish_wrapper(){
+    tmp_qemu_wrapper=$(mktemp -tdp /tmp gf-vsmt.XXXXXX)
+    qemu_wrapper=${tmp_qemu_wrapper}/qemu-wrapper.sh
+	cat <<-'EOF' > "${qemu_wrapper}"
+	#!/bin/bash -
+	exec qemu-system-ppc64 "$@" -machine pseries,accel=kvm:tcg,vsmt=8,cap-fwnmi=off
+	EOF
+    chmod +x "${qemu_wrapper}"
+    export LIBGUESTFS_HV="${qemu_wrapper}"
```

Comment:
I think we need to set this variable at *runtime* too, not just build time.  (I mean, this file is the build of coreos-assembler itself, but we need to set it when coreos-assembler is performing a build of FCOS/RHCOS)

Reply:
So if we do the above of installing the file literally, then I think what we want to do is set this environment variable like:
`export LIBGUESTFS_HV=/usr/lib/coreos-assembler/libguestfs-ppc64-wrapper.sh`
inside `cmdlib.sh` or so?

Does that make sense?

Reply:
Ahh right.  Though, now I'm confused why `cosa build` doesn't also fail...oh, but I bet it would if we went to build another artifact; we only run libguestfs when we go to change `ignition.platform.id`.

Hmm.  It's kind of tempting to by default produce an "unstamped" artifact and force libguestfs into the default build path too.

(And deduplicate the libguestfs environment variables between build and kola)

Reply:
I totally forgot the `export` won't persist in the pod creation. Thanks @cgwalters and @jlebon